addons/mail/static/src/models/thread/thread.js

```diff
@@ -1077,6 +1077,16 @@ function factory(dependencies) {
 
         /**
          * @private
+         * @returns {boolean}
+         */
+        _computeHasNeedaction() {
+            // TODO: this could rely on the message_needaction_counter when fixed
+            return this.needactionMessages.length > 0;
+        }
```

Comment:
don't need this compute, just directly use this.needactionMessagesAsOriginThread.length in the other compute